{
  "parameter(s)": [
    "f",
    "o",
    19,
    32
  ],
  "test_compare_lengths_with_large_string": [
    "passed",
    "",
    0.00023
  ],
  "test_property_of_function": [
    "failed",
    "AssertionError: assert 'l' not in 'vvsdVypmSEt...DerZhWWzfObY'\n  'l' is contained here:\n    vvsdVypmSEtpBCmeaUDbCqrJWPlveZuSWlSACafdGxauUDerZhWWzfObY\n  ?                           +",
    0.00058
  ],
  "test_repeat_char": [
    "failed",
    "IndexError: string index out of range",
    0.00032
  ]
}